package/MDAnalysis/converters/OpenMM.py

```diff
@@ -74,9 +74,6 @@ class OpenMMSimulationReader(base.SingleFrameReaderBase):
 
 
     .. versionadded:: 2.0.0
-    .. versionchanged:: 2.4.0
-      removed type and mass guessing (guessing takes place now inside universe)
-
```

Comment:
Why remove this? Isn't it a valuable bit of documentation? This is an actual change to the OpenMM converter.

Reply:
I already added version changes to OpenMMParser.py, I didn't make any extra changes to OpenMM.py that's why I removed it